paymentsheet-example/src/androidTest/java/com/stripe/android/test/core/PlaygroundTestDriver.kt

```diff
@@ -1038,6 +1038,7 @@ internal class PlaygroundTestDriver(
 
     internal fun teardown() {
         application?.unregisterActivityLifecycleCallbacks(activityLifecycleCallbacks)
+        currentActivity[0] = null
```

Comment:
Unrelated nit: Should `currentActivity` just be a `var` instead of an array?

Reply:
Yup! `@Volatile var`  would do just as good! I can follow up.